src/content/docs/network-performance-monitoring/setup-performance-monitoring/advanced-config.mdx

```diff
@@ -0,0 +1,687 @@
+          </tr>
+
+          <tr>
+            <td>mib_profile_dir</td>
+            <td></td>
+            <td>Directory to find curated MIB profiles</td>
+          </tr>
+        </tbody>
+      </table>
+    </Collapser>
+    <Collapser
+        id="snmpv3-config"
+        title="SNMPv3 optional configuration"
+      >
+        Here's an example of `SNMPv3` configuration section for the `snmp-base.yaml` file:
+```yaml
+  default_v3:
+    user_name: userNamev3
+    authentication_protocol: MD5
+    authentication_passphrase: authPassPrivacy
+    privacy_protocol: AES256
+    privacy_passphrase: passPrivacy
+```
+
+        <table>
+        <thead>
+          <tr>
+            <th style={{ width: "200px" }}>Option name</th>
+            <th>Required</th>
+            <th>Description</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>user_name</td>
+            <td>✓</td>
+            <td>User name for SNMPv3 authentication</td>
+          </tr>
+
+          <tr>
+            <td>authentication_protocol</td>
+            <td>✓</td>
+            <td>SNMPv3 authentication protocol. The possible values are `NoAuth`, `MD5`, or `SHA`</td>
+          </tr>
+
+          <tr>
+            <td>authentication_passphrase</td>
+            <td></td>
+            <td>SNMPv3 authentication passphrase</td>
+          </tr>
+
+          <tr>
+            <td>privacy_protocol</td>
+            <td>✓</td>
+            <td>SNMPv3 privacy protocol. The possible values are `AuthNoPriv`, `DES`, `AES`, `AES192`, `AES256`, `AES192C`, or `AES256C`</td>
+          </tr>
+
+          <tr>
+            <td>privacy_passphrase</td>
+            <td></td>
+            <td>SNMPv3 privacy passphrase</td>
+          </tr>
+
+          <tr>
+            <td>context_engine_id</td>
+            <td></td>
+            <td>SNMPv3 context engine ID</td>
+          </tr>
+
+          <tr>
+            <td>context_name</td>
+            <td></td>
+            <td>SNMPv3 context name</td>
+          </tr>
+        </tbody>
+      </table>
+  </Collapser>
+</CollapserGroup>
+
+## The **snmp_flow_only** attribute [#snmp_flow_only-attribute]
+
+There are several supported configurations available for running **ktranslate** against devices for both SNMP and network flow data collection. The `snmp_flow_only` attribute on both a **ktranslate** docker container level and in the `devices` section of the `snmp-base.yaml` file allows you to get different results:
+
+- If you have multiple containers collecting both SNMP and network flow data with the **ktranslate** docker image, you'll define your devices in the `snmp-base.yaml` file, following the standards for SNMP polling and they will be automatically matched to their respective flows based on their `device_ip` matching the sampling IP for the flow device. 
```

Comment:
I'd remove the "you'll"

Reply:
I'd make this two sentences, it's very long. I'll break it after "SNMP polling".